### DISCUSSION TITLE
为什么有幻觉问题呢

### DISCUSSION THREAD

**Discussion by songge25** [OleehyO/TexTeller#73] [Q&A]
你好，我在预训练模型上，使用了20w自己的公式数据训练。出现了幻觉问题，会识别出不存在的选项

![image2024-7-25_10-48-56](https://github.com/user-attachments/assets/59947895-3db9-4a74-bc65-235a581237c8)

ocr：
将 2.05 \times 10^{-3} 用小数表示(    )
A.
 0.000205
B.
 0.00205
C.
 0.00205

**Comment by OleehyO** [comment]
幻觉问题还是挺正常的，根据我的经验一般只要把数据量加大，把数据增强做的更丰富就能有效解决。

**Comment by songge25** [comment]
好的 感谢指点！
